chessboard/olegbet/chessboard.go

```diff
@@ -0,0 +1,54 @@
+package chessboard
+
+// Declare a type named Rank which stores if a square is occupied by a piece - this will be a slice of bools
+
+type Rank []bool
```

Comment:
```suggestion
type Rank [8]bool
```
I'd declare the size here.